docs/crds/CRD-docs-for-docs-repo.md

```diff
@@ -7,6 +7,12 @@
 
 ## policies.kubewarden.io/v1
 
+This linter directive is added to avoid issues with the
```

Comment:
I'm not sure these phrases should be part of this file, if there's a linter that gives a false positive, we should disable it for the specific file with https://github.com/kubewarden/kubewarden-controller/blob/main/.golangci.yml#L11.